rllib/utils/explorations/__init__.py

```diff
@@ -0,0 +1,6 @@
+from ray.rllib.utils.explorations.exploration import Exploration
```

Comment:
Please rename this package from `explorations` => `exploration`.

Reply:
done